mbed_host_tests/host_tests_conn_proxy/conn_primitive_serial.py

```diff
@@ -84,6 +87,9 @@ def reset_dev_via_serial(self, delay=1):
 
     def read(self, count):
         """! Read data from serial port RX buffer """
+        # TIMEOUT: Since read is called in a loop, wait for self.timeout period before calling serial.read(). See
+        # comment on serial.Serial() call above about timeout.
+        sleep(self.timeout)
```

Comment:
Won't this have a pretty bad impact on timing tests? Now we only have a granularity of 200ms in which we can assign timestamps, which will cause significant timing drifts. I'm not sure we want to add this call to `sleep` quite yet.